package.json

```diff
@@ -61,6 +62,10 @@
 		"eslint-plugin-import": "^2.25.4",
 		"file-loader": "^6.2.0",
 		"h2o2": "^8.2.0",
+		"jsdoc": "^3.6.10",
+		"jsdoc-advanced-types-plugin": "github:tomalec/jsdoc-advanced-types-plugin#add/return-support",
```

Comment:
💅  ❓  I guess this fork should be potentially added as well to the tool repo?

Reply:
Deffinitely